### PR TITLE
Docs for initial release of Forms 14

### PR DESCRIPTION
**Pull request by AndyButland**
## Description

Added documentation on breaking changes and updates to how custom fields are registered.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [X] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Forms 14

## Deadline (if relevant)

From 30th May.

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @AndyButland thank you for the PR 🙌 

We will make sure to review it and merge it in from 30th of May 👍

File: 14/umbraco-forms/developer/extending/adding-a-fieldtype.md [hunk 1]
```diff
@@ -155,19 +193,105 @@ The area aliases for the other provider types are as follows:
 - Recordset actions - `formRecordSetActions`
 - Workflows - `formProviderWorkflows`
 
-The `View` attribute defines the client-side view used when rendering a preview of the field in the form's designer. Umbraco Forms ships with a number of these, found in a virtual path of `App_Plugins\UmbracoForms\backoffice\Common\SettingTypes\`.
+The `View` property indicates a property editor UI used for editing the setting value. You can use a built-in property editor UI, one from a package, or a custom one registered with your solution.  The default value if not provided is `Umb.PropertyEditorUi.TextBox`, which will use the standard Umbraco text box property editor UI.
+
+You may optionally want to register a settings value converter. This is a client-side, manifest registered component, that converts between the setting value required for the editor and that persisted with the form definition. A converter defines three methods:
```
Comment (eshanrnh) [inline 14/umbraco-forms/developer/extending/adding-a-fieldtype.md:198]:
```suggestion
You may want to consider registering a settings value converter. This is a client-side component that is registered in the manifest. It converts the setting value required for the editor and the value persisted with the form definition. A converter defines three methods:
```

File: 14/umbraco-forms/release-notes.md [hunk 2]
```diff
@@ -15,77 +15,18 @@ If you are upgrading to a new major version, you can find information about the
...
-* Resolved an out of range exception when a condition hides all fields on the final page of a multi-page form.
-* Fixed issue with swapping between plain and rich text when configuring the post-submission message [#1145](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1145).
-
-#### [**13.0.0**](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(December 14th 2023)**
-
-* Compatibility with Umbraco 13
-  * See full details of breaking changes under the [version specific upgrade guide](upgrading/version-specific.md#version-13).
-* Ran registered server-side file validators on file uploads.
+* Compatibility with Umbraco 14
+  * See full details of breaking changes under the [version specific upgrade guide](upgrading/version-specific.md).
```
Comment (eshanrnh) [inline 14/umbraco-forms/release-notes.md:23]:
```suggestion
  * See full details of breaking changes under the [Version-specific Upgrade Guide](upgrading/version-specific.md).
```

File: 14/umbraco-forms/release-notes.md [hunk 3]
```diff
@@ -15,77 +15,18 @@ If you are upgrading to a new major version, you can find information about the
...
 
 ## Umbraco.Forms.Deploy
 
-#### [13.0.1] (March 19th 2024)
+#### **14.0.0** **(May 30th 2024)**
 
-* Added migrator to support changes in Form field prevalues when importing from older Umbraco versions
+* Compatibility with Umbraco 13, Forms 13 and Deploy 13.
```
Comment (eshanrnh) [inline 14/umbraco-forms/release-notes.md:29]:
```suggestion
* Compatibility with Umbraco 13, Forms 13, and Deploy 13.
```

File: 14/umbraco-forms/upgrading/version-specific.md [hunk 4]
```diff
@@ -5,62 +5,46 @@ description: >-
 
 # Version Specific Upgrade Notes
 
-This page covers specific upgrade documentation for when migrating to major 13 of Umbraco Forms.&#x20;
+This page covers specific upgrade documentation for when migrating to major 14 of Umbraco Forms.&#x20;
```
Comment (eshanrnh) [inline 14/umbraco-forms/upgrading/version-specific.md:8]:
```suggestion
This page provides specific upgrade documentation for migrating to Umbraco Forms version 14.
```

File: 14/umbraco-forms/upgrading/version-specific.md [hunk 5]
```diff
@@ -5,62 +5,46 @@ description: >-
...
-For reference, the full details are listed here:
+Version 14 contains a number of breaking changes, primarily due to the new backoffice introduced in Umbraco 14. The details are listed here:
 
 #### **Behavior**
 
-* The configuration option `UseSemanticFieldsetRendering` was removed and considered as `true`. This means the improved, semantic fieldset rendering is now used in the default theme for all installations.
-* The `ExecuteWorkflowAsync` configuration option was removed.
+* A new management API has been introduced at `umbraco/forms/management/api`.
+* The root of the existing delivery API used for headless/AJAX solutions has moved to `umbraco/forms/delivery/api`.
+* The HTML helper `RenderUmbracoFormDependencies` no longer renders the promises polyfill, which isn't needed in modern browsers.
```
Comment (eshanrnh) [inline 14/umbraco-forms/upgrading/version-specific.md:26]:
```suggestion
* The HTML helper `RenderUmbracoFormDependencies` no longer renders the promises polyfill, which is not needed in modern browsers.
```

File: 14/umbraco-forms/upgrading/version-specific.md [hunk 6]
```diff
@@ -5,62 +5,46 @@ description: >-
...
+* Server-side registration of content apps has been removed as this is now a client-side concern.
+* Creation of custom field, workflow and other provider types remains primarily a server-side task. However they no longer require the provision of AngularJS views and controllers. Instead these reference registered client-side manifests. Read more on this under [extending Umbraco Forms](../developer/extending/README.md).
```
Comment (eshanrnh) [inline 14/umbraco-forms/upgrading/version-specific.md:28]:
```suggestion
* Creation of custom fields, workflow, and other provider types remains primarily a server-side task. However, they no longer require the provision of AngularJS views and controllers. Instead, these reference registered client-side manifests. For more information, see the [extending Umbraco Forms](../developer/extending/README.md) article.
```

**Review by eshanrnh — APPROVED** [review]
The PR looks good to me, @AndyButland 🙌

I've made a suggestion to write a sentence clearly. Can you take a look at it? I've incorporated the minor suggestions myself. 💪

File: 14/umbraco-forms/developer/extending/adding-a-fieldtype.md (hunk 1 above)
Comment (AndyButland) [reply 14/umbraco-forms/developer/extending/adding-a-fieldtype.md:198]: Improves on what I had for sure, but to be precise I think should be:

```
You may want to consider registering a settings value converter. This is a client-side component that is registered in a manifest. It converts between the setting value required for the editor and the value persisted with the form definition. A converter defines three methods:
```

Comment (eshanrnh) [reply 14/umbraco-forms/developer/extending/adding-a-fieldtype.md:198]: Thanks, I'll make the change then 💪

Comment (eshanrnh) [inline 14/umbraco-forms/developer/extending/adding-a-fieldtype.md:198]:
```suggestion
You may want to consider registering a settings value converter. This is a client-side component that is registered in a manifest. It converts between the setting value required for the editor and the value persisted with the form definition. A converter defines three methods:
```

**Comment by eshanrnh** [comment]
Good to go 💪I'll merge this on 30th of May.

**Pull request merged**